oc_mirror.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
Would prefer to see `-u` and `-o pipefail` also set. It's too easy to misspell the name of an environment variable.

Reply:
Fixed